libraryapi/pergamum.py

```diff
@@ -109,21 +109,23 @@ def convert_dados_marc_to_record(dados_marc: DadosMarc) -> Record:
 
 
 class PergamumDownloader:
+    """Represents a serialized Record object ready to be used"""
```

Comment:
Na verdade não representa isso não. É uma classe utiliza uma conexão com pergamum e utiliza dos conversores para gerar o formato desejado

Reply:
Handle the connection to the Pergamum Web Service and transform the retrieved data to several representations

Reply:
Se ainda não for isso, você muda depois.